GameMethod.py

```diff
@@ -25,16 +29,19 @@ def initialize_method(self):
     def play_one_turn(self, user_input):
         pass
 
+    @abstractmethod
+    def is_game_finished(self):
```

Comment:
Hey, I thought about it. I agree, it makes more sense. but i didn't move it because this method contains other elements which i think don't belong to GameState. My idea was to make it a very simple class (maybe it should be an enum?)